--- frameworks/base/Android.mk.orig	2017-07-13 11:33:58.241849882 +0800
+++ frameworks/base/Android.mk	2017-07-13 11:32:25.683284715 +0800
@@ -127,6 +127,8 @@
 	core/java/android/os/IPowerManager.aidl \
 	core/java/android/os/IRemoteCallback.aidl \
 	core/java/android/os/IVibratorService.aidl \
+	core/java/android/os/ILed_demo_service.aidl \
+	core/java/android/os/ICallback.aidl \
 	core/java/android/service/wallpaper/IWallpaperConnection.aidl \
 	core/java/android/service/wallpaper/IWallpaperEngine.aidl \
 	core/java/android/service/wallpaper/IWallpaperService.aidl \
